apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationStatusForm/ApplicationStatusForm.tsx

```diff
@@ -209,12 +210,7 @@ export const ApplicationStatusForm = ({
             <Submit
               color="primary"
               isSubmitting={isSubmitting}
-              text={intl.formatMessage({
-                id: "OunUSG",
```

Comment:
You'll need to remove `OunUSG` from `apps/web/src/lang/fr.json`.

Reply:
Removed in [trim unused translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5799/commits/0621b4a23be5afc6da9c07cdf7b12a6caf88acd0)